CI broken -- Tensor Flow example test failing

We've seen a bunch of errors like this:

```shell
--- FAIL: TestExampleTensorFlow (142.90s)
	require.go:153: /home/travis/gopath/src/github.com/pachyderm/pachyderm/src/server/examples/examples_test.go:63
	require.go:155: No error is expected but got commit GoT_generate/GoT_generate:1795f2cfd68f4d1288e37a72314817f9:0 was cancelled
```

After some digging, the train job is failing (silently! the following pipeline notices the expected output file isn't there ... so I need to make sure this job exiting fails properly).

The logs from the train job:

```shell
$pachctl get-logs e57fa81e4f1cf85aa5fe4734d963114c
0 | Epoch: 1 Learning rate: 1.000
0 | Traceback (most recent call last):
0 |   File "ptb_word_lm.py", line 435, in <module>
0 |     tf.app.run()
0 |   File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/platform/app.py", line 30, in run
0 |     sys.exit(main(sys.argv[:1] + flags_passthrough))
0 |   File "ptb_word_lm.py", line 308, in main
0 |     train()
0 |   File "ptb_word_lm.py", line 351, in train
0 |     verbose=True)
0 |   File "ptb_word_lm.py", line 270, in run_epoch
0 |     state = m.initial_state.eval()
0 | AttributeError: 'tuple' object has no attribute 'eval'
0 | cp: cannot stat '/data/ptb.ckpt*': No such file or directory
```

The logs from the generate job are below. The output of the `GoT_train` pipeline can't be found, indicating that job failed somehow. The only job reported as failed is the generate job:

```shell
$pachctl list-job
ID                                 OUTPUT                                            STARTED             DURATION            STATE               
aa826f8aba88c2bcfcefda22949f6560   GoT_generate/e131694cc9f2487aae9594f64297ebed/0   3 minutes ago       2 seconds           failure    
e57fa81e4f1cf85aa5fe4734d963114c   GoT_train/3b045f712ef14950b88b002eea389801/0      3 minutes ago       3 seconds           success   
$pachctl get-logs aa826f8aba88c2bcfcefda22949f6560
0 | cp: cannot stat '/pfs/GoT_train/ptb.ckpt*': No such file or directory
0 | Traceback (most recent call last):
0 |   File "ptb_word_lm.py", line 435, in <module>
0 |     tf.app.run()
0 |   File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/platform/app.py", line 30, in run
0 |     sys.exit(main(sys.argv[:1] + flags_passthrough))
0 |   File "ptb_word_lm.py", line 314, in main
0 |     generate(word_to_id, id_to_word)
0 |   File "ptb_word_lm.py", line 372, in generate
0 |     f = open(os.path.join(FLAGS.model_path_prefix, "ptb.ckpt"), "r")
0 | IOError: [Errno 2] No such file or directory: '/training_data/ptb.ckpt'
0 | Error from exec: exit status 1
```

--- a/doc/examples/tensor_flow/Dockerfile
+++ b/doc/examples/tensor_flow/Dockerfile
@@ -1,4 +1,4 @@
-FROM tensorflow/tensorflow
+FROM tensorflow/tensorflow:0.10.0
 MAINTAINER jdoliner@pachyderm.io
 
 # Install FUSE

